### ISSUE TITLE
Add example with Tailwind

### ISSUE THREAD

**Issue by deadcoder0904** [neg4n/next-api-og-image#2]
Awesome work! Love how simple it is. I've checked every existing solution & this feels simplest.

I would love to have something like [Bannerbear](https://www.bannerbear.com/) which creates dynamic previews & I love to style in Tailwind so would love it if I can get a template like Tailwind.

**Comment by neg4n** [comment]
Hey! Thanks for the feedback 😊

Using Tailwind in `next-api-og-image` would look like that
```js
import { withOGImage } from 'next-api-og-image'

export default withOGImage({
  html: ({ myQueryParam }) =>
    `
      <html>
        <head>
          <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
        </head>
        <body>
          <h1 class="flex flex-row text-6xl text-blue-300">${myQueryParam}<span>💻</span></h1>
        </body>
      </html>
    `,
})
```

I've tested it and it seems to work as expected :0 Request to `http://localhost:3000/api/basic-tailwind?myQueryParam=im%20using%20Tailwind!!` looks like this

![image](https://user-images.githubusercontent.com/57688858/135998790-8246a647-b039-4bd7-ba11-c099c8f6e722.png)


I will add this example to README and `example/` Next.js project. Thanks for the suggestion! 🤟🏻

**Comment by deadcoder0904** [comment]
Knew it could be done with `link`.

I'd also probably like to add separate `html` file templates which are not hardcoded, probably using `fs` module. Would that work too?

**Comment by neg4n** [comment]
Hmm, I doubt the separate `html` templates along with `fs`. Im thinking about 2 reasons why:
1. Placement of query params in the template - `next-api-og-image` is stupid and only passes the query forward to `html` function in configuration. The *closest* solution would be creating something like `my-template.html.js` with
	  ```js
	  export default function myTemplate({ superCoolQueryParam }) {
	    return `<h1>${superCoolQueryParam}</h1>`
	  }
	  ```
	  then passed to the API Route using **withOGImage**
	  ```js
	  import { withOGImage } from 'next-api-og-image'
	  import myTemplate from './my-template.html'
	  
	  export default withOGImage({ html: myTemplate })
	  ```
2. Most likely, there will be same issues as loading custom fonts while loading such things by `fs` module. Although it works on plain vercel serverless functions - I believe it does not in Next.js and i think its caused by invalid paths caused by bundling the source to .next directory.

**Comment by deadcoder0904** [comment]
The 1st solution is good. Is there any way to preview the HTML we'll be displaying then so we can get live feedback like React's HMR?

I mean I'd like if it's possible to have something like https://og-image.vercel.app/ in development mode as I want to integrate this library into my blog but would love to see how the templates look so I can create multiple dynamic ones.

Right now, it feels like I'd have to see the screenshot or create another project that I work on HTML & then copy-paste the template into this which is bad DX :(

**Comment by neg4n** [comment]
> > I mean I'd like if it's possible to have something like og-image.vercel.app in development mode as I want to integrate this library into my blog but would love to see how the templates look so I can create multiple dynamic ones.
> 
> Sure! The development mode HTML inspect seems reasonable, will create separate issue for it and implement it today

Hey @deadcoder0904! This is now implemented (#4) (#5) 🎉 
Let me know if there is anything that you would to suggest in this topic or eventually
feel free to close the issue if the subject got exhausted.

Cheers 🤗 Igor

**Comment by deadcoder0904** [comment]
Thanks @neg4n, great work. I'll open a new one if I have any or re-open this one if it's related.
